common/config/environment.go

```diff
@@ -45,6 +46,11 @@ const (
 
 	// Testing workload namerNamespace
 	DDTestingWorkloadDeployParamName = "deploy"
+
+	// Dogstatsd namespace
+	DDDogstatsdDeployParamName        = "deploy"
+	DDDogstatsdAPIKeyParamName        = "apiKey"
```

Comment:
Do you think that we should just reuse `ddagent:apiKey` or go for the solution that you suggested in the comment below?

Reply:
I think that just reusing `ddagent:apiKey` is simpler.
And I cannot see any limitation it would bring. (Why would we want the agent and dogstatsd standalone to target two different orgs ?)
Let’s go for the simplest solution.

Reply:
Changed :+1: